Silicon/Intel/IntelSiliconPkg/Feature/Flash/SpiFvbService/SpiFvbServiceCommon.c

```diff
@@ -568,6 +568,25 @@ GetVariableFvInfo (
     return;
   }
 
+  //
+  // GetVariableFlashNvStorageInfo () only reports regular variable region information,
+  // if platform implemented an additional NVS region following the regular variable region,
+  // the both region size should be included as overall NVS region size.
+  // Example NVS flash map for such usage model:
+  // Note: PcdFlashNvStorageAdditionalSize is equal to platform PcdFlashFvNvStorageEventLogSize

```

Comment:
How about:
Note: PcdFlashNvStorageAdditionalSize is often PcdFlashFvNvStorageEventLogSize in legacy use
          and both should be deprecated in favor of explicitly defined FV regions.